examples/braket_features/Noise_models/Noise_models_on_Amazon_Braket.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>It might be helpful to spell-out that this saves the file locally on the client (for instance your laptop or your Braket Notebook Instance). It is generally unclear to users when things are persisted locally versus S3.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/152/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>